app/views/patients/show.html.erb

```diff
@@ -8,21 +8,29 @@
 <%= react_component('subject/DownloadMonitoreeExcel', { authenticity_token: form_authenticity_token, patient: @patient }) %>
 <% end %>
 
-<%= react_component('patient/PatientPage', { current_user: current_user, patient: @patient, group_members: @group_members, patient_id: @patient.user_defined_id_statelocal || '', dashboardUrl: '/public_health', authenticity_token: form_authenticity_token, hideBody: current_user.can_modify_subject_status?, jurisdictionPath: @patient.jurisdiction[:path] }) %>
+<%= react_component('patient/PatientPage', {
+                      current_user: current_user,
+                      patient: @patient,
+                      dependents: @dependents_exclude_hoh,
+                      patient_id: @patient.user_defined_id_statelocal || '',
+                      dashboardUrl: '/public_health',
+                      authenticity_token: form_authenticity_token,
+                      hideBody: current_user.can_modify_subject_status?,
+                      jurisdiction_path: @patient.jurisdiction[:path]
+                    }) %>
 
 <% if current_user.can_modify_subject_status? %>
 <div class="card mx-2 mt-4 mb-4 card-square">
   <h5 class="card-header">Monitoring Actions</h5>
-  <%= react_component('subject/MonitoringStatus', {
+  <%= react_component('subject/monitoring_actions/MonitoringActions', {
                         current_user: current_user,
                         user_can_transfer: current_user.can_transfer_patients?,
                         authenticity_token: form_authenticity_token,
-                        has_group_members: @group_members.exists?,
-                        in_a_group: @all_group_members.count > 1,
+                        has_dependents: @dependents_exclude_hoh.exists?,
+                        in_household_with_member_with_ce_in_exposure: @household_members_with_ce_in_exposure_excludes_patient.exists?,
```

Comment:
I'm seeing the following error when going to view a monitoree's page and I believe it's due to this line (it just highlights the wrong thing in the error message). You can't call `exists?` on an array.
![Screen Shot 2020-10-29 at 12 11 29 PM](https://user-images.githubusercontent.com/11698457/97601442-8d5d4200-19e0-11eb-9c7f-74eccd295b90.png)

Reply:
I'm not seeing this error - can you test on the new PR and let me know if it's still happening?